indico/modules/rb/templates/emails/reservations/reminders/upcoming_occurrence.html

```diff
@@ -18,6 +18,10 @@
             </li>
         {%- endfor -%}
     </ul>
+    {%- if occurrences | length > 1 -%}
+        If you wish to cancel all occurrences, please <a href="{{ reservation.external_details_url }}">cancel the
```

Comment:
I'd wrap after "please" to avoid wrapping in the middle of a tag's content (especially of a tag that's not in its own line). [same in the block below]

Since it's HTML whitespace doesn't really matter here anyway. In any case, the next line has 4 spaces indentation too much 👾